### PR TITLE
Dashboard launch blog post

### PR DESCRIPTION
**Pull request by ccarfi**
dashboard launch blog post

### CODE REVIEW

**Comment by ccarfi** [comment]
post ready! if everything looks ok, feel free to squash & merge.

File: unlock-protocol-com/blog/dashboard-redesign.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+![Screenshot 2022-09-28 at 13.16.57.png](/images/blog/dashboard-redesign/Screenshot_2022-09-28_at_13.16.57.png)
+
+### Manage lock page
+
+![manage-lock-mobile.png](/images/blog/dashboard-redesign/manage-lock-mobile.png)
+
+## See the new Dashboard now
+
+Ready to check it out? **[Try out the new Dashboard](https://app.unlock-protocol.com/dashboard)**.
```
Comment (kalidiagne) [inline unlock-protocol-com/blog/dashboard-redesign.md:120]:
```suggestion
Ready to check it out? **[Try out the new Dashboard](https://app.unlock-protocol.com/locks)**.
```

Comment (kalidiagne) [reply unlock-protocol-com/blog/dashboard-redesign.md:120]: @julien51 based on the discussion in https://github.com/unlock-protocol/unlock/pull/9802 we eventually need to change with `/locks` path or keep both url

**Review by kalidiagne — APPROVED** [review]

File: unlock-protocol-com/blog/dashboard-redesign.md (hunk 1 above)
Comment (julien51) [reply unlock-protocol-com/blog/dashboard-redesign.md:120]: ok thx

**Pull request merged**